src/components/package/ProgramPackageCollectionBlock.tsx

```diff
@@ -176,7 +176,7 @@ const ProgramPackageCollectionBlock: React.VFC<{
               options={sortOptions}
               defaultValue={sort}
               onChange={event => setSort(event.target.value)}
-              disabled={programPackage.length === 0}
+              disabled={search !== '' && programPackageCounts > 0 && programPackage.length === 0}
```

Comment:
The naming of  `programPackageCounts` may be confusing, it looks similar to `programPackage.length`. Perhaps `totalProgramPackageCounts` or `filteredProgramPackage`.length would be clearer.

Reply:
Adjusted.